docs/cloud/cloud-authentication/cloud-auth-revoke-key.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Revoke Key
+layout: default
+parent: Manage API Keys
+grand_parent: Cloud
+nav_order: 5
+---
+
+# How do I revoke an API key in FeatureBase Cloud?
+{: .no_toc }
+
+Learn how to revoke a FeatureBase Cloud API key.
+
+{% include page-toc.md %}
+
+## Warnings
+
+* Revoked keys can no longer be used. 
+* API calls using a revoked key will be rejected. 
```

Comment:
The second and third bullet seem to be referring to the same thing and can be rejigged as follows:

```
* API calls that are not running will be rejected after the key is revoked
```

This then means we can economize on words in the last bullet by doing this:

```
After a key is revoked, API calls will:
* complete as normal if already in progress
* be revoked if the call has not yet started
```